tests/dispatch_io.c

```diff
@@ -240,13 +240,19 @@ test_io_read_write(void)
 {
 	const char *path_in = "/usr/share/dict/words";
 	char path_out[] = "/tmp/dispatchtest_io.XXXXXX";
-	const size_t siz_in = 1024 * 1024;
 
 	int in = open(path_in, O_RDONLY);
 	if (in == -1) {
 		test_errno("open", errno, 0);
 		test_stop();
 	}
+    struct stat sb;
```

Comment:
alignment is not right, we use tabs.

The vim settings to use for our coding rules are mostly:

```vim
set cinoptions=
set cinoptions+=Ls
set cinoptions+=:0,=1s
set cinoptions+=t0
set cinoptions+=+2s,(2s,u2s,W2s
set cinoptions+=j1,J1,m1

au FileType d,c,cpp,objc setlocal tw=80 ts=4 sw=4 noet noignorecase